apps/antalmanac/src/components/AppBar/ImportStudyList.tsx

```diff
@@ -183,12 +183,13 @@ class ImportStudyList extends PureComponent<ImportStudyListProps, ImportStudyLis
                         Import
                     </Button>
                 </Tooltip>
-                <Dialog 
-                    open={this.state.isOpen} 
-                    onClose={() => this.setState(
-                        { isOpen: false, studyListText: ''}, 
-                        async () => {document.removeEventListener('keydown', this.enterEvent, false)}
-                    )}
+                <Dialog
+                    open={this.state.isOpen}
+                    onClose={() =>
+                        this.setState({ isOpen: false, studyListText: '' }, async () => {
+                            document.removeEventListener('keydown', this.enterEvent, false);
+                        })
+                    }
```

Comment:
Regardless of what the linter says, I think that this is just less readable. @EricPedley, what do you say should take precedent?